src/plugins/media-analysis/media-analysis-utils.js

```diff
@@ -54,6 +65,65 @@ export function createDefaultContent(t) {
   };
 }
 
+export function exportChaptersToCsv(chapters) {
+  return [
+    'startPosition,title,color,text',
+    ...chapters.map(chapter => [
+      chapter.startPosition.toString(),
+      `"${chapter.title.replace(/"/g, '""')}"`,
+      `"${chapter.color.replace(/"/g, '""')}"`,
+      `"${chapter.text.replace(/"/g, '""')}"`
+    ].join(',')),
+    ''
+  ].join('\n');
+}
+
+export function importChaptersFromCsv(csvStringOrFile) {
+  return new Promise((resolve, reject) => {
+    parseCsv(csvStringOrFile, {
+      header: true,
+      skipEmptyLines: true,
+      error: error => reject(Object.assign(new Error(), error)),
```

Comment:
Is `error` not instance of `Error` then here, to just `reject(error)` directly? 🤔

Reply:
Exactly. Unfortunately they are plain objects (that contain - amongst others - a `message` property), so I had to assign those properties to a "real" error instance while not loosing all the error details.